wazo_agid/handlers/outgoingfeatures.py

```diff
@@ -96,7 +107,9 @@ def _set_caller_id(self) -> None:
             if selected_caller_id == _ANONYMOUS_CALLER_ID:
                 self._set_anonymous()
             else:
-                objects.CallerID.set(self._agi, selected_caller_id)
+                self._agi.set_variable(
+                    'WAZO_SELECTED_CALLER_ID_TO_FORMAT', selected_caller_id
+                )
```

Comment:
Could we expose the information with a more generic variable name e.g. simply `WAZO_SELECTED_CALLER_ID` ?

Reply:
`TO_FORMAT` seems to prescribe what to do with the information, but I don't see why that's necessary.